packages/calcite-design-tokens/support/token-transformer/styleDictionary/formatter/utils/formatExtraOutput.ts

```diff
@@ -49,7 +49,8 @@ export function formatExtraOutput(
                 const m = ensureIfArray(outputObject[`${output}.${args.platform}`]);
                 const cssProps = m.map((o) => `${o}`.replaceAll("$", "--"));
                 return Array.isArray(m)
-                  ? `${output === "light" ? `:root {${cssProps.join("")}}` : ""
+                  ? `${
```

Comment:
This just a formatting change?

Reply:
Yes, I think it got formatted when the latest from `main` got merged.